### ISSUE TITLE
What should the blink directive be called?

### ISSUE THREAD

**Issue by jansensan** [jansensan/ng-useless#1]
I would like to use `<blink>`, however I do not want to collide with a possible remain of an actual valid `<blink>` tag.

Should I go with `<ng-blink>`, or do you have other suggestions?

**Comment by mgnsharon** [comment]
it's bad practice to use ng as a namespace.  What happens if the angular team creates one?

**Comment by jansensan** [comment]
Then how about these?
- `<blink-v2>`
- `<blinking>`
- `<blink-redux>`

I'm not a fan of the first one myself...

**Comment by mgnsharon** [comment]
- `<uu-blink>`
- `<blinky>`

We need to create a namespace and rename the project to __-useless

**Comment by jansensan** [comment]
Good point, however that is a different issue. This conversation should be moved to another thread. Please stay on subject.

**Comment by jansensan** [comment]
@csharon I like `<blinky>`, or even `<blinking>`. Is `<blinker>` a word? I like going with something like this better than `<cah-blink>` for example.
